templates/myaccount/form-login.php

```diff
@@ -104,7 +104,7 @@
 
 			<p class="woocommerce-FormRow form-row">
 				<?php wp_nonce_field( 'woocommerce-register', 'woocommerce-register-nonce' ); ?>
-				<button type="submit" class="woocommerce-Button button" name="register" value="<?php esc_attr_e( 'Register', 'woocommerce' ); ?>"><?php esc_html_e( 'Register', 'woocommerce' ); ?></button>
+				<button type="submit" class="woocommerce-button button woocommerce-form-register__submit" name="register" value="<?php esc_attr_e( 'Register', 'woocommerce' ); ?>"><?php esc_html_e( 'Register', 'woocommerce' ); ?></button>
```

Comment:
Please keep `woocommerce-Button` for backwards compatibility.

Reply:
Just let as `woocommerce-Button woocommerce-button button woocommerce-form-register__submit` and it will be fine.